docroot/themes/humsci/su_humsci_theme/patterns/molecules/vertical-card/vertical-card.html.twig

```diff
@@ -18,7 +18,7 @@
       </div>
     {% endif %}
     {% if button %}
-      <a class="decanter-button" href="\"#\"">{{ button }}</a>
-  {% endif %}
-</div>
+      <div class="vertical-card__button">{{ button }}</div>
```

Comment:
I like this approach better as a button is likely an atom that could be slotted in here.

Reply:
do you mean you like it as is or you'd like it better as `<button class="vertical-card__button">{{ button }}</button>`?

Reply:
The field is better than the hardwired link. 👍